internal/api/mocks/schema_service.go

```diff
@@ -81,7 +105,16 @@ func (_m *SchemaService) DeleteVersion(ctx context.Context, namespace string, sc
 func (_m *SchemaService) Get(ctx context.Context, namespace string, schemaName string, version int32) (*schema.Metadata, []byte, error) {
 	ret := _m.Called(ctx, namespace, schemaName, version)
 
+	if len(ret) == 0 {
+		panic("no return value specified for Get")
+	}
+
```

Comment:
can we return an error instead of panic, since this method returns an error anyways?